### DISCUSSION TITLE
Why is an anchor tag needed inside a Link component?

### DISCUSSION THREAD

**Discussion by pradow** [vercel/next.js#29763] [Help]
I am going through the next tutorial and I am confused by this step 

https://nextjs.org/learn/basics/navigate-between-pages/link-component

It is telling me to add an a tag with no href attribute inside a Link component but it seems that the Link component works without having an a tag inside of it.

Can someone let me know if I should be including this a tag in a Link component?

**Comment by jeffcad** [comment]
[This thread](https://github.com/vercel/next.js/discussions/17139) has the answer.
